rmgpy/data/thermo.py

```diff
@@ -1443,7 +1463,7 @@ def correct_binding_energy(self, thermo, species):
         comments = []
         for element in 'CHON':
             if normalized_bonds[element]:
-                change_in_binding_energy = self.delta_atomic_adsorption_energy[element].value_si * normalized_bonds[element]
+                change_in_binding_energy = delta_atomic_adsorption_energy[element].value_si * normalized_bonds[element]
                 thermo.H298.value_si += change_in_binding_energy
                 comments.append(f'{normalized_bonds[element]:.2f}{element}')
         thermo.comment += " Binding energy corrected by LSR ({})".format('+'.join(comments))
```

Comment:
As Katrin noted, we shouldn't append this comment if the delta is zero (we're scaling from Pt111 to Pt111). Probably we should just skip the whole method if `metal_to_scale_from == metal_to_scale_to`.

Reply:
I was under the impression it wouldn't call `correct_binding_energy` if it didn't need to scale?

Reply:
I'll fix this, but do you think we should be calling this if there is no change?

Reply:
In general we should avoid calling it if we don't need to. But does 1756a659d233404b933f90f0a77822247b21b78d address it well enough?

Reply:
I think we should append the comment with what metal we are scaling from.